cd/helm/fogus/templates/netappdjango-deployment.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     io.kompose.service: netappdjango
   name: netappdjango
 spec:
-  replicas: 1
+  replicas: {{ .Values.capif_hostname }}
```

Comment:
capif_hostname maybe is wrong variable to setup number of replicas of this service. it should be {{ .Values.app_replicas }}